src/modules/app/components/SearchDialog.tsx

```
@@ -0,0 +1,175 @@
+            <ul>
+              {results.map((note, index) => (
+                <li
+                  key={note.id}
+                  className={cn(
+                    "relative flex flex-col items-start cursor-pointer gap-2 rounded-sm px-2 py-1.5 mx-1.5 my-1 text-sm outline-hidden select-none hover:bg-accent hover:text-accent-foreground",
+                    index === activeIndex && "bg-accent text-accent-foreground",
+                  )}
+                  onClick={() =>
+                    handleSelect(() => router.push(`/notes/${note.id}`))
+                  }
+                >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)